src/components/utils/__tests__/TimeUtils.test.js

```diff
@@ -1,91 +1,66 @@
-import moment from 'moment'
+import dayjs from 'dayjs'
+import MockDate from 'mockdate'
 
 import TimeUtils from '../TimeUtils'
 
-// setup mocked date generation
-let now = null
 beforeAll(() => {
-    // whenever moment() is invoked 2017-03-22 will be taken as date
-    Date.now = jest.fn(() => new Date(Date.UTC(2017, 2, 22)).valueOf())
-    now = moment()
+    // whenever dayjs() is invoked 2017-03-22 will be taken as date
+    MockDate.set(new Date(Date.UTC(2017, 2, 23)))
```

Comment:
The day in the comment and in the original source code is `22`, not `23`.

```suggestion
    MockDate.set(new Date(Date.UTC(2017, 2, 22)))
```

Also, this was going to be fixed when merging #160, but maybe you want also to do it here, since it's a one line change: remove the `Date.UTC` (see #160 for details about why).

Here's a suggested line change that covers both issues:

```suggestion
    MockDate.set(new Date(2017, 2, 22))
```

Reply:
Sure, this sounds like a good solution, I'll do this later tonight, if possible. Thanks